lib/jsonapi/authorization.rb

```diff
@@ -7,6 +7,14 @@
 
 module JSONAPI
   module Authorization
+    def user_context(context)
+      if JSONAPI::Authorization.configuration.pundit_user.is_a?(Symbol)
+        context[JSONAPI::Authorization.configuration.pundit_user]
+      else
+        JSONAPI::Authorization.configuration.pundit_user.call(context)
+      end
+    end
+    module_function :user_context
```

Comment:
Do you think that this is the best place for this method? One option would be to move it to `Authorization::Configuration` class, so that calling it would look like this:

```rb
user_context = JSONAPI::Authorization.configuration.user_context(context)
```

It's a little bit more typing but it has that feeling it is a part of the configuration. What do you think?